### PR TITLE
Update j9vm build dependencies for CRIU

### PR DESCRIPTION
**Pull request by tajila**
Update j9vm build dependencies

Fixes: https://github.com/eclipse-openj9/openj9/issues/14926

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
Jenkins compile xlinuxcriu jdk11

**Comment by pshipton** [comment]
The commit title should have more context, i.e. Update j9vm build dependencies for criu

**Comment by tajila** [comment]
Jenkins test sanity xlinuxcriu jdk11

File: runtime/criusupport/CMakeLists.txt [hunk 1]
```diff
@@ -42,6 +42,7 @@ target_link_libraries(j9criu
 		j9util
 		j9thr
 		dl
+		criu
```
Comment (keithc-ca) [inline runtime/criusupport/CMakeLists.txt:45]: Unless there's a reason not to do so, can you insert `criu` before `dl`?

**Comment by keithc-ca** [comment]
jenkins compile xlinuxcriu jdk11

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**